pandas/core/ops.py

```diff
@@ -709,6 +709,15 @@ def wrapper(left, right, name=name, na_op=na_op):
 
         left, right = _align_method_SERIES(left, right)
 
+        if is_timedelta64_dtype(left) and isinstance(right, pd.DatetimeIndex):
+            # GH#13905
+            # Defer to DatetimeIndex/TimedeltaIndex operations where timezones
+            # are handled carefully.
+            result = op(pd.TimedeltaIndex(left), right)
+            return construct_result(left, result,
+                                    index=left.index, name=left.name,
+                                    dtype=result.dtype)
```

Comment:
should `name` be passed using `com._maybe_match_name` here?  Not sure what the convention is.

Reply:
yes